pacman/utilities/file_format_converters/convert_to_java_machine.py

```diff
@@ -0,0 +1,133 @@
+from pacman.utilities import file_format_schemas
+from spinn_utilities.progress_bar import ProgressBar
+from spinn_machine.router import Router
+
+import json
+
+CHIP_HOMOGENEOUS_CORES = 18
+
+
+class ConvertToJavaMachine(object):
+    """ Converter from memory machine to java machine
+    """
+
+    __slots__ = []
+
+    def __call__(self, machine, file_path):
+        """
```

Comment:
fill in

Reply:
Fill in what? There's docs here. You might want _more_ docs… but that's just for being needlessly wordy unless you actually go to the effort to define what the actual format produced is for the docs, and that's quite a bit of work for something that almost nobody will ever read. Ever.